pkgs/development/libraries/sope/default.nix

```diff
@@ -0,0 +1,42 @@
+{ gnustep, lib, fetchFromGitHub , libxml2, openssl_1_1
+, openldap, mysql, libmysqlclient, postgresql ? null }: with lib; gnustep.stdenv.mkDerivation rec {
+  pname = "sope";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    owner = "inverse-inc";
+    repo = pname;
+    rev = "SOPE-${version}";
+    sha256 = "02bnnxxb9ay5md12rc2ydxqq9sg5nm3jrpp98jwih1hi59l0qlm7";
+  };
+
+  nativeBuildInputs = [ gnustep.make ];
+  buildInputs = [ gnustep.base libxml2 openldap openssl_1_1 mysql libmysqlclient ]
+    ++ optional (postgresql != null) postgresql;
+
+  postPatch = ''
+    # Exclude NIX_ variables
+    sed -i 's/grep GNUSTEP_/grep ^GNUSTEP_/g' configure
+  '';
+
+  preConfigure = ''
+    export DESTDIR="$out"
+  '';
+
+  configureFlags = [ "--prefix=" "--disable-debug" "--enable-xml" "--with-ssl=ssl" "--enable-mysql" "--enable-openldap" ]
+    ++ optional (postgresql != null) "--enable-postgresql";
+
+  # Yes, this is ugly.
+  preFixup = ''
+    cp -rlPa $out/nix/store/*/* $out
+    rm -rf $out/nix/store
```

Comment:
This should be `rm -rf $out/nix`, otherwise it leaves an empty `nix` dir behind.